app/assets/css/tailwind.css

```diff
@@ -4,3 +4,8 @@
  */
 @import "tailwindcss";
 @source "../../templates";
+
+a {
+  color: blue;
+  text-decoration: underline;
+}
```

Comment:
I had to do this to stop myself from going crazy finding links, happy to remove this